nautobot/extras/tables.py

```diff
@@ -266,19 +268,35 @@ class ConfigContextSchemaValidationStateColumn(tables.Column):
     """
 
     def __init__(self, validator, data_field, *args, **kwargs):
+        kwargs.setdefault("orderable", False)
+        kwargs.setdefault("accessor", None)
+        kwargs.setdefault("exclude_from_export", True)
+        kwargs.setdefault("empty_values", ())
+
         super().__init__(*args, **kwargs)
+
         self.validator = validator
         self.data_field = data_field
 
     def render(self, *, record):  # pylint: disable=arguments-differ  # tables2 varies its kwargs
-        data = getattr(record, self.data_field)
+        data = getattr(record, self.data_field, None)
+        # don't do import here, just doing to demonstrate the fix
+        from jsonschema import Draft7Validator, SchemaError
+
         try:
-            self.validator.validate(data)
-        except JSONSchemaValidationError as e:
-            # Return a red x (like a boolean column) and the validation error message
-            return render_boolean(False) + format_html('<span class="text-danger">{}</span>', e.message)
+            if hasattr(record, "local_config_context_schema"):
```

Comment:
@sirisha-bode this logic should be moved to the views.py. I am not exactly sure how/where, but you should be able to take a crack at it.